user.php

```diff
@@ -0,0 +1,108 @@
+<?
+
+
+class User{
+
+	public $db;
+
+	private $states = [
+		'NSW',
+		'ACT',
+		'VIC',
+		'QLD',
+		'TAS',
+		'WA'
+	]
+
+	function __construct()
+	{
+		$this->db = new mysqli('127.0.0.1', 'root', 'password', 'users');
+
+	}
+
+	function check_Postcode($postcode) {
+		//.... call some external api
+		$res = external_api_call();
+		if (!$res) {
+			throw new \Exception("invalid post code");
```

Comment:
1) This could be namespace imported in the begging of the class

``` use Exception```

and then 

``` throw new Exception("invalid post code");```

Also, a more specific exception should be used instead of a generic one.